core/storage/trie/polkadot_trie/polkadot_trie_cursor.cpp

```diff
@@ -169,36 +162,36 @@ namespace kagome::storage::trie {
   }
 
   outcome::result<void> PolkadotTrieCursor::prev() {
-    BOOST_ASSERT_MSG(false, "Not implemented yet");
     return Error::METHOD_NOT_IMPLEMENTED;
   }
 
   common::Buffer PolkadotTrieCursor::collectKey() const {
     KeyNibbles key_nibbles;
-    for (auto &node_idx : last_visited_child_) {
-      auto &node = node_idx.parent;
+    for (auto const &node_idx : last_visited_child_) {
+      auto const &node = node_idx.parent;
```

Comment:
Minor. We use const-qualifier before type.
```suggestion
    for (const auto &node_idx : last_visited_child_) {
      const auto &node = node_idx.parent;
```

Reply:
We actually have 32 "**auto const**" matches in the project, so I guess it's kinda optional.

Reply:
Upd: Okay, agreed on const auto.